### PR TITLE
Commented out

### PR DESCRIPTION
**Pull request by aikoldolubaev**

### CODE REVIEW

File: src/components /todolist/TodoList.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ class  TodoList extends Component{
                     return  (
                       <li key={id}><input type="checkbox"/>
                       {item}
-                      <span onClick={()=> this.onDelete(id)} className="delete">X</span>
+                      {/* <span onClick={()=> this.onDelete(id)} className="delete">X</span> */}
```
Comment (baigazieva) [inline src/components /todolist/TodoList.js]: Hey, don't comment out please..

**Review by baigazieva — CHANGES_REQUESTED** [review]
Please, read my comments

File: src/components /todolist/TodoList.js (hunk 1 above)
Comment (aikoldolubaev) [reply src/components /todolist/TodoList.js]: Ok, thank you for reminding, totally missed that

Comment (aikoldolubaev) [reply src/components /todolist/TodoList.js]: Hey, just fixed it. Thank you, it was a good catch.

**Review by baigazieva — APPROVED** [review]
Good job!

**Pull request merged**